pybandits/strategy.py

```diff
@@ -250,12 +297,13 @@ class MultiObjectiveBandit(Strategy):
         Number of objectives to be solved by the bandit (n_objectives must be >= 1).
     """
 
+    # TODO: remove kwargs
```

Comment:
I tried, but we cannot remove it. It's because of Pydantic checks, we are forced to put kwargs (unless investigate more and find a more elegant solution)